### PR TITLE
wip: Viewing absentee ballot status

### PR DESCRIPTION
**Pull request by joshing-dev**

### CODE REVIEW

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+@Composable
+fun NonAbsentee() {
+  Surface(
+    shape = MaterialTheme.shapes.medium,
+    elevation = 2.dp
+  ) {
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+      Text(
+        text = "You are not an absentee voter. Click below to view how to register for absentee voting.",
```
Comment (scottschmitz) [inline app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt:25]: strings.xml

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        style = MaterialTheme.typography.h1,
+        modifier = Modifier.padding(16.dp)
+      )
+      val uriHandler = LocalUriHandler.current
+      PrimaryButton(
+        text = "Register Absentee",
```
Comment (scottschmitz) [inline app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt:31]: strings.xml

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+@Composable
+fun AbsenteeApplicationReceived(registration: VoterRegistration) {
+  Surface(
+    shape = MaterialTheme.shapes.medium,
+    elevation = 2.dp) {
+    Column {
+      Text(
+        text = "You are registered to vote as absentee!",
```
Comment (scottschmitz) [inline app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt:23]: strings.xml

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        style = MaterialTheme.typography.h1,
+        modifier = Modifier.padding(16.dp))
+      Text(
+        text = "Application last received: ${registration.absenteeApplicationReceived ?: "N/A"}",
```
Comment (scottschmitz) [inline app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt:27]: strings.xml

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.fueledbycaffeine.mivote.ui.theme.MIVoteTheme
+import io.michiganelections.api.model.VoterRegistration
+
+
+@Composable
+fun AbsenteeApplicationReceived(registration: VoterRegistration) {
+  Surface(
```
Comment (scottschmitz) [inline app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt:18]: I believe you wanted to just use `Card` here? That should automatically use the "medium" shape and give the default elevation.

**Comment by scottschmitz** [comment]
The background also _moves_ when you toggle between the two screens... On your screen Michigan is getting cut off at the top.

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt (hunk 1 above)
Comment (joshing-dev) [reply app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt:25]: Fixed in #19

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt (hunk 2 above)
Comment (joshing-dev) [reply app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/NonAbsentee.kt:31]: Fixed in #19

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt (hunk 4 above)
Comment (joshing-dev) [reply app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt:27]: Fixed in #19

File: app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt (hunk 5 above)
Comment (joshing-dev) [reply app/src/main/java/com/fueledbycaffeine/mivote/ui/voter/ballot/AbsenteeApplicationReceived.kt:18]: Fixed in #19

**Pull request closed without merging**